osu.Game/Overlays/Settings/Sections/General/UpdateSettings.cs

```diff
@@ -143,9 +143,8 @@ private async Task checkForUpdates()
             }
             finally
             {
-                // This sequence allows the notification to be immediately dismissed.
-                checkingNotification.State = ProgressNotificationState.Cancelled;
```

Comment:
this code changed, can you explain why

Reply:
Replaced with a helper method which also doesn't change the state to avoid the notification showing as "cancelled" for a single frame as I saw during testing.